dvc/remote/webdavs.py

```diff
@@ -0,0 +1,15 @@
+from .webdav import RemoteWEBDAV
+from dvc.scheme import Schemes
+
+
+class RemoteWEBDAVS(RemoteWEBDAV):
+    scheme = Schemes.WEBDAVS
+
+    def gc(self):
+        raise NotImplementedError
+
+    def list_cache_paths(self, prefix=None, progress_callback=None):
```

Comment:
This can be implemented using the WebDAV `PROPFIND` method, but may not be worth it for now unless we want to support `gc` for WebDAV remotes.